src/renderer/components/ClusterPage.vue

```diff
@@ -92,6 +92,10 @@ export default {
     lensLoaded: function() {
       console.log("lens loaded")
       this.lens.loaded = true;
+      this.lens.webview.getWebContents().on('new-window', (e, url) => {
```

Comment:
I know that we are only using a one or two month old version of electron. But https://www.electronjs.org/docs/breaking-changes#removed-webviewgetwebcontents seems to say that this will not be possible if we ever choose to upgrade. Would seeing if it is possible to use the recommended newer version be a good idea?

Reply:
@Nokel81 good catch!

Reply:
Yes, definitely 🍻 . Changed to implementation.